Resideo_eshopping/resideo_eshopping/.flutter-plugins-dependencies

```diff
@@ -1 +1 @@
-{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"firebase_auth","dependencies":["firebase_core"]},{"name":"firebase_core","dependencies":[]},{"name":"firebase_database","dependencies":["firebase_core"]},{"name":"firebase_storage","dependencies":["firebase_core"]},{"name":"flutter_full_pdf_viewer","dependencies":[]},{"name":"flutter_pdfview","dependencies":[]},{"name":"flutter_plugin_pdf_viewer","dependencies":["path_provider"]},{"name":"image_cropper","dependencies":[]},{"name":"image_picker","dependencies":[]},{"name":"path_provider","dependencies":[]},{"name":"screen","dependencies":[]},{"name":"shared_preferences","dependencies":[]},{"name":"sqflite","dependencies":[]},{"name":"video_player","dependencies":[]}]}
\ No newline at end of file
+{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"firebase_auth","dependencies":["firebase_core"]},{"name":"firebase_core","dependencies":[]},{"name":"firebase_database","dependencies":["firebase_core"]},{"name":"firebase_storage","dependencies":["firebase_core"]},{"name":"flutter_full_pdf_viewer","dependencies":[]},{"name":"flutter_pdfview","dependencies":[]},{"name":"flutter_plugin_pdf_viewer","dependencies":["path_provider"]},{"name":"geolocator","dependencies":["google_api_availability","location_permissions"]},{"name":"google_api_availability","dependencies":[]},{"name":"image_cropper","dependencies":[]},{"name":"image_picker","dependencies":[]},{"name":"location_permissions","dependencies":[]},{"name":"path_provider","dependencies":[]},{"name":"shared_preferences","dependencies":[]},{"name":"sqflite","dependencies":[]},{"name":"video_player","dependencies":[]}]}
```

Comment:
Auto generated files shouldn't be commited
//This is a generated file; do not edit or check into version control